### DISCUSSION TITLE
March 2022 SP-API Release Announcement

### DISCUSSION THREAD

**Discussion by timmyscj** [amzn/selling-partner-api-models#3294] [General]
I am pleased that the [new release](https://developer-docs.amazon.com/sp-api/changelog/march-2022-sp-api-release-announcement) allows for more historical data. I haven't been able to successfully use the new reportTypes.

Has anyone had success using any of the following new reportTypes?
GET_VENDOR_SALES_REPORT and GET_VENDOR_TRAFFIC_REPORT (previously GET_VENDOR_SALES_DIAGNOSTIC_REPORT)
GET_VENDOR_INVENTORY_REPORT (previously GET_VENDOR_INVENTORY_HEALTH_AND_PLANNING)

I receive a FATAL processingStatus when I request the new report. Swapping the reportType with the previous report works as expected, which leads me to believe there is an issue on Amazon's end.

Another frustration encountered while using this API...

**Comment by timmyscj** [comment]
> I am pleased that the [new release](https://developer-docs.amazon.com/sp-api/changelog/march-2022-sp-api-release-announcement) allows for more historical data. I haven't been able to successfully use the new reportTypes.
> 
> Has anyone had success using any of the following new reportTypes? GET_VENDOR_SALES_REPORT and GET_VENDOR_TRAFFIC_REPORT (previously GET_VENDOR_SALES_DIAGNOSTIC_REPORT) GET_VENDOR_INVENTORY_REPORT (previously GET_VENDOR_INVENTORY_HEALTH_AND_PLANNING)
> 
> I receive a FATAL processingStatus when I request the new report. Swapping the reportType with the previous report works as expected, which leads me to believe there is an issue on Amazon's end.
> 
> Another frustration encountered while using this API...

Has anyone been able to successfully use any of the new vendor apis?

**Reply by ANarayanan-Cervello** [reply]
Hi @timmyscj , the new API (GET_VENSOR_SALES_REPORT) required you to specify some additional "reportOptions". Are you sure that its been provided?
